include/motis/tag_lookup.h

```diff
@@ -18,7 +18,9 @@ struct tag_lookup {
   nigiri::source_idx_t get_src(std::string_view tag) const;
   std::string_view get_tag(nigiri::source_idx_t) const;
   std::string id(nigiri::timetable const&, nigiri::location_idx_t) const;
-  std::string id(nigiri::timetable const&, nigiri::rt::run_stop) const;
+  std::string id(nigiri::timetable const&,
+                 nigiri::rt::run_stop,
+                 nigiri::event_type const) const;
```

Comment:
```suggestion
                 nigiri::event_type) const;
```

`const` says that the implementation doesn't modify but is irrelevant for the caller in case of value parameters -> should be omitted in the header (only relevant int the source so the compiler can check)

Reply:
ah yes should have remembered the style guide...